bindings/python/iota_sdk/client/_utils.py

```diff
@@ -44,15 +44,23 @@ def hex_to_bech32(self, hex_str: HexStr, bech32_hrp: str) -> str:
             'bech32Hrp': bech32_hrp
```

Comment:
Can you also make it optional for this function? (can't comment further up)

Reply:
Done

Reply:
Thanks